x-pack/solutions/security/plugins/security_solution/docs/testing/test_plans/detection_response/prebuilt_rules/prebuilt_rules_common_info.md

```diff
@@ -114,6 +114,9 @@ Terminology related to UI and UX:
 
 - **CTA**: "call to action", usually a button, a link, or a callout message with a button, etc, that invites the user to do some action.
 
+Terminology related to the rule customization and upgrade workflow:
```

Comment:
Why is it related to `rule customization and upgrade workflow` while the description below only mentions `rule customization`? It looks like it could be moved to `Terminology related to prebuilt rule customization` section.